### DISCUSSION TITLE
can I forcely use "baseline" profile when turning on "/gfx-h264"?

### DISCUSSION THREAD

**Discussion by coconutLatte** [FreeRDP/FreeRDP#8795] [Q&A]
Environment:
- code: FreeRDP v2.10.0
- client: Ubuntu 20.04
- server: Windows-Server 2019

cmd line: `xfreerdp /u:Administrator /p:1234Qwer /v:10.0.5.9 /gfx-h264 /app:"||Google Chrome" /app-cmd:--user-data-dir=c:\tmp\chrome\16782276832`

I turn on `/gfx-h264` on and dump the video message from Windows-Server which profile is "Main" while I want "baseline"

here is ffprobe out 
![image](https://user-images.githubusercontent.com/68467481/224956334-d119c429-172c-4cab-a5cc-bc72f55c68bd.png)

Is there anything I can do to make the profile to "baseline"? I except it to be "baseline" as it received from Windows-Server.

**Comment by akallabeth** [comment]
long story short, check your server OS documentation ;)
client side you can only turn it on or off

**Reply by coconutLatte** [reply]
Hi, thanks for reply.
by the way, how can I turn it on client side?

**Reply by akallabeth** [reply]
well, `h264` support is activated by default. (manually with`/gfx:avc444` and/or `/network:auto`)

**Reply by coconutLatte** [reply]
I've already catch `h264` stream. The question is its profile is `main` while what I want is `baseline`.
could it be changed by setting `/network:auto` or other client settings?

**Reply by akallabeth** [reply]
no. you can check [MS-RDPEGFX](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpegfx/da5c75f9-cd99-450c-98c4-014a496942b0) yourself, maybe I´ve missed something, but there is no way the client can force anything here.
